card/mod/core/set/all/codename.rb

```diff
@@ -16,7 +16,8 @@ def codename
 private
 
 def validate_codename_permission
-  return if Auth.always_ok?
+  return if Auth.always_ok? || Auth.as_id == creator_id
```

Comment:
this is so that Urs can add codenames for his metrics. Not a super well-considered change, but probably not one that will cause many problems.